### PR TITLE
[FrontEnd]enable pdpd ops layer_norm, gelu, fill_any_like, cumsum, tanh, matmul_v2 for BERT

### PR DESCRIPTION
**Pull request by luo-cheng2021**
### Details:
 - *Enable Following Ops*
     - [layer_norm](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/static/nn/layer_norm_en.html#layer-norm)
         - The limitation: 'begin_norm_axis' should be greater than 0 coming from the [code](https://github.com/PaddlePaddle/Paddle/blob/cec36ea6ff16fda90c1a004c6e043cd9b2096a2a/paddle/fluid/operators/layer_norm_op.cc#L176)
     - [gelu](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/gelu_en.html#gelu)
     - [fill_any_like](https://github.com/PaddlePaddle/Paddle/blob/35ef41800fe0aacf80b4f2f9b6272a169cb226e3/python/paddle/tensor/creation.py#L223): no document, use source code instead
     - [cumsum](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/cumsum_en.html#cumsum)
     - [tanh](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/tanh_en.html#tanh)
     - [matmul_v2](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/matmul_en.html)

### Tickets:
 - *63556*

### CODE REVIEW

**Comment by ceciliapeng2011** [comment]
@jane-intel @itikhono @nosovmik Please have a look

File: ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+namespace op {
+NamedOutputs fill_any_like(const NodeContext& node) {
+    auto x = node.get_ng_input("X");
+    auto dtype = node.get_attribute<ngraph::element::Type>("dtype", element::undefined);
+    auto value = node.get_attribute<float>("value");
+    if (dtype == element::undefined) {
+        // when type does not define, use the input type
+        dtype = x.get_element_type();
+    }
+    auto supported_type = {element::i32, element::i64, element::f16, element::f32, element::f64};
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp:20]: const please

File: ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    bool valid_type = std::any_of(supported_type.begin(), supported_type.end(), [dtype](const element::Type& type) {
+        return dtype == type;
+    });
+    PDPD_ASSERT(valid_type, "fill_any_like only supports i32, i64, f16, f32, f64");
+    auto value_node = opset6::Constant::create(dtype, {1}, {value});
+    auto shape_node = std::make_shared<opset6::ShapeOf>(x);
+
+    return node.default_single_output_mapping({std::make_shared<ngraph::opset6::Broadcast>(value_node, shape_node)},
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp:28]: better opset8 ?

File: ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+#include <ngraph/opsets/opset6.hpp>
+#include <node_context.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs layer_norm(const NodeContext& node) {
+    using namespace ngraph::opset6;
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp:13]: opset8

File: ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp [hunk 4]
```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp:5]: I am thinking of a common header for paddle/op folder, to include opsetXX.hpp, and namespace, how do you think?

File: ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp (hunk 1 above)
Comment (luo-cheng2021) [reply ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp]: Updated.

File: ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp (hunk 2 above)
Comment (luo-cheng2021) [reply ngraph/frontend/paddlepaddle/src/op/fill_any_like.cpp]: Updated

File: ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp (hunk 3 above)
Comment (luo-cheng2021) [reply ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp]: Updated.

File: ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp (hunk 4 above)
Comment (luo-cheng2021) [reply ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp]: Added default_opset.hpp like ONNX mode. I think we may need another PR to sync the modification if it's acceptable.

Comment (ceciliapeng2011) [reply ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp]: great. Let's do it after our bert PRs merged. :)

**Review by mvafin — APPROVED** [review]

File: ngraph/frontend/paddlepaddle/src/op/cumsum.cpp [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs cumsum(const NodeContext& node) {
+    auto x = node.get_ng_input("X");
```
Comment (ilyachur) [inline ngraph/frontend/paddlepaddle/src/op/cumsum.cpp:14]: Looks like we don't change `x` and other variables, so I think it can be const.

Comment (luo-cheng2021) [reply ngraph/frontend/paddlepaddle/src/op/cumsum.cpp]: Updated.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**